encoding/text/marshal.go

```diff
@@ -461,7 +461,6 @@ func (enc *Encoder) marshalList(elem schema.Type, l capnp.List) error {
 	default:
 		return fmt.Errorf("unknown list type %v", elem.Which())
 	}
-	return nil
```

Comment:
Revert this line; unrelated?

Reply:
Yeah, I updated the commit message with the following sentence:
`Also delete unreachable return nil in marshal.go` I could also add this to the description or would you prefer a separate PR?

Reply:
Makes sense. This is fine.